.github/workflows/reformat_yaml_files.yml

```
@@ -0,0 +1,18 @@
+name: Reformat Yaml Files
+
+on: [push]
```

```suggestion
on: [pull_request]
```

This workflow should only be triggered on new PRs.